frontend/src/routes/+page.svelte

```diff
@@ -1,2 +1,39 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<script lang="ts">
+	export let form;
+</script>
+
+<div class="flex min-h-screen items-center justify-center bg-gray-100 px-4 py-12 sm:px-6 lg:px-8">
+	<div class="w-full max-w-md space-y-8">
+		<div>
+			<h2 class="mt-6 text-center text-3xl font-extrabold text-gray-900">Input URL</h2>
+		</div>
+		<form method="post" class="mt-8 space-y-6">
+			<div class="-space-y-px rounded-md shadow-sm">
+				<div>
+					<label for="url" class="sr-only">URL</label>
+					<input
+						id="url"
+						name="url"
+						type="text"
+						required
+						class="relative block w-full appearance-none rounded-none rounded-t-md border border-gray-300 px-3 py-2 text-gray-900 placeholder-gray-500 focus:z-10 focus:border-indigo-500 focus:outline-none focus:ring-indigo-500 sm:text-sm"
+						placeholder="https://example.com"
```

Comment:
nit: we are asking the user to input twitter post URLs here, so the placeholder should be something similar to `https://twitter.com/<some-handler>/status/<id-number>`

Reply:
Thanks! updated in this commit.
- 524b5c81cd16d8bd0df99dd1922ae56d25e0b34d